frontend/game/play/play.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
-define(function() {
-    var helper = require('./helper.js');
+define(function(require) {
```

Comment:
Maybe this is why tests failed.

Reply:
nah, that's a separate issue I have to fix